site/app/templates/course/CourseMaterials.twig

```diff
@@ -17,6 +17,10 @@
                     tabindex="0">Set All Release Dates</a>
             <a onclick="newUploadCourseMaterialsForm()" class="btn btn-primary key_to_click" tabindex="0">Upload Course Materials</a>
         </div>
+    {% else %}
+        <div class="action-buttons" style="float: right; margin-bottom: 20px;">
+            <a onclick='setCookie("foldersOpen",openAllDivForCourseMaterials());' class="btn btn-primary key_to_click" tabindex="0">Open/Close All Folders</a>
+        </div>
```

Comment:
Instead of duplicating code, it might be better to move the these lines outside the if-statement, and use the if-statement to control the visibility of the release date and course materials upload buttons.

Reply:
I believe my new commit fixes this issue.